### PR TITLE
4.3: Fix SLE SP wrong references

### PR DESCRIPTION
**Pull request by juliogonzalez**
# Description

Fix SLE SP wrong references. We still had references to SP5 as base OS.

# Target branches

- master: https://github.com/uyuni-project/uyuni-docs/pull/2771
- 4.3: This PR

# Links
- https://github.com/uyuni-project/uyuni-docs/issues/2733

### CODE REVIEW

File: parameters.yml [hunk 1]
```diff
@@ -174,11 +174,13 @@ asciidoc:
   - attribute: sp-base
     value: SP4
   - attribute:  sp-version
-    value: SP5 #use for normal text
+    value: SP4 #use for normal text
   - attribute: sp-vert
```
Comment (juliogonzalez) [inline parameters.yml:178]: I wonder why we need `sp-vert` and `sp-version-l` if the value is 100% the same but... OK :-D

File: parameters.yml [hunk 2]
```diff
@@ -174,11 +174,13 @@ asciidoc:
   - attribute: sp-base
     value: SP4
   - attribute:  sp-version
-    value: SP5 #use for normal text
+    value: SP4 #use for normal text
```
Comment (juliogonzalez) [inline parameters.yml:177]: Same as below, not clear to me why we need `sp-base` and `sp-version.

File: parameters.yml (hunk 1 above)
Comment (keichwa) [reply parameters.yml:178]: > I wonder why we need `sp-vert` and `sp-version-l` if the value is 100% the same but... OK :-D

long time ago, I defined `sp-version-l`.  It some incarnation of `sp-vert` or `sp-ver` in long format.  I do not remember exactly the values.  Maybe, "SP 5" vs. "Service Pack 5" or just "5" vs. "SP 5" ;)  We often do not know how to properly phase out a variable no longer needed.

File: parameters.yml (hunk 2 above)
Comment (keichwa) [reply parameters.yml:177]: One for the SP version SUSE Manager is based on (4.3 SP4), and the other latest supported SLE client (4.3 supports SP5)
IIRC, we want to switch to sp-server and sp-client for clarity.  I proposed it, Joseph agreed, but I hesitated to change it actually.

**Review by keichwa — APPROVED** [review]
Your changes look syntactically correct, but I'm not an expert with version numbers in general.  We'll need at least a second tic here.

File: parameters.yml (hunk 2 above)
Comment (juliogonzalez) [reply parameters.yml:177]: So what should the be right value then? Whatever entity is used for clients should have a clear name for it.. or at least a clear comment. Does this need to stay as SP5?

File: parameters.yml (hunk 1 above)
Comment (jcayouette) [reply parameters.yml:178]: lol not sure why I did not notice this before. We should keep these simple for sure. Phase it out! Phase it out! :)

**Review by jcayouette — COMMENTED** [review]
See comments.

**Comment by juliogonzalez** [comment]
> See comments.

@jcayouette I don't understand what to exactly about them. Do I need to rollback some values to SP4, or are they OK as they are in the PR?

If a refactor is needed, that would be for another PR :-)

**Comment by jcayouette** [comment]
@juliogonzalez Your changes look good to me. Thank you!

**Review by jcayouette — APPROVED** [review]

**Pull request merged**